patientsearch/api.py

```diff
@@ -183,15 +207,33 @@ def external_search(resource_type, methods=["GET"]):
     external_search_bundle = external_request(token, resource_type, request.args)
     local_fhir_patient = sync_bundle(token, external_search_bundle)
 
+    # TODO: is there a PHI safe 'id' for the user (in place of email)?
```

Comment:
We can have Keycloak bake different fields into the JWT, I think keycloak user ID *should* be possible

Reply:
Oohh - I like that @ivan-c .  Doesn't belong in this PR, but let's go that route if possible.